Sources/MapboxCoreNavigation/EHorizon/RoadName.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct RoadName {
+
+    public let name: String
+
+    public let isShielded: Bool
```

Comment:
I suppose this property means the road name is actually a road number that can be displayed in a generic shield? For consistency with MapboxDirections, we should expose either a `name` that’s actually a name or a `code` (aka “ref”) in the event that `MapboxNavigationNative.RoadName.isShielded` is true.